New Look and Feel for Proof of Humanity

santisiri opened the pull request:
The new UI has an improved look and feel for the Open & Democratic Proof of Humanity. Changes include:
- New logo and icon that works well at all resolutions.
- Dark mode by default user experience
- Improved navigation of buttons & upload wizard
- Links to all of the resources created exclusively by the DAO 
- Ready to work with Open & Democratic PoH v2 fork.

santisiri committed:
feat: dark mode

santisiri committed:
feat: dao links

santisiri committed:
feat: accordeon button ocean color

santisiri committed:
feat: ethermail

santisiri committed:
feat: rounded panels

santisiri committed:
fix: nicer logo

santisiri committed:
feat: better icons

santisiri committed:
fix: ubi logo

santisiri committed:
fix: icon display

santisiri committed:
fix: header button design

santisiri committed:
fix: softer is better

santisiri committed:
fix: button arrangement adapts to mobile

santisiri committed:
feat: wizard edit

santisiri committed:
feat: texts need more emojis

santisiri committed:
fix: translation

santisiri committed:
fix: tweaks in ui for profile upload

santisiri committed:
fix: tooltip colors

santisiri committed:
fix: camera controls

santisiri committed:
fix: video uploader tool

santisiri committed:
Merge pull request #4 from OpenProofOfHumanity/style

New Look and Feel for the UI

santisiri committed:
fix: new logo

santisiri committed:
Merge pull request #5 from OpenProofOfHumanity/style

New Logo

bilinkis committed:
fix: merged latest changes from master

bilinkis commented on components/accordion.js:
```diff
@@ -55,23 +57,26 @@ function AnimatedAccordionItemHeading({ expanded, children, ...rest }) {
-          backgroundImage({ colors: { accent, accentComplement } }) {
-            return `linear-gradient(90deg, ${accent} 0%, ${accentComplement} 100%, ${accentComplement} 100%)`;
-          },
+          backgroundImage: `linear-gradient(90.13deg, var(--theme-ui-colors-accent, ${accColor}) 49.89%, var(--theme-ui-colors-accentComplement, #517aea) 93.37%)`,
```
Not sure I like how these colors look like. I'd probably stick to the orange color identity.

bilinkis reviewed (commented):
Never used Ethermail.io, but it looks to be beta and not open yet for everyone.

bilinkis commented on components/accordion.js:
```diff
@@ -55,23 +57,26 @@ function AnimatedAccordionItemHeading({ expanded, children, ...rest }) {
-          backgroundImage({ colors: { accent, accentComplement } }) {
-            return `linear-gradient(90deg, ${accent} 0%, ${accentComplement} 100%, ${accentComplement} 100%)`;
-          },
+          backgroundImage: `linear-gradient(90.13deg, var(--theme-ui-colors-accent, ${accColor}) 49.89%, var(--theme-ui-colors-accentComplement, #517aea) 93.37%)`,
```
Not sure I like how these colors look like. I'd probably stick to the orange color identity.

santisiri replied:
Those colors are on the same gamma that we are using across the board and brings in a 2nd set of colors that can be used to contextualize better the information specifically about challenges. I will keep on working on the "challenges" menu, specially as we connect to a Court provider on Polygon and the networks we'll deploy for the v2.

bilinkis commented on components/account-settings-popup.js:
```diff
@@ -90,10 +89,14 @@ export default function AccountSettingsPopup({
-          <Settings />
+        <IconButton sx={{ cursor: "pointer", marginTop: "3px", width: "40px" }}>
+          <Image
+            src="/images/ethereum.svg"
```
Not sure this icon reflects what the menu is for, but the menu is pretty useless altogether, so might as well delete it.

santisiri replied:
It's really a menu about handling your Ethereum connection to the dapp (not really about Settings). That's why we use the ETH icon. That said: I will keep polishing that menu there on subsequent versions.

bilinkis commented on components/header.js:
```diff
@@ -224,30 +224,22 @@ function DesktopNavbar() {
+          <Col span={15} className="poh-header-item">
             <NextLink href="/" as="/">
               <Link className="poh-header-text" variant="navigation">
                 {t("header_profiles")}
```
Design looks good but not centered within the navigation bar

santisiri replied:
It's attached to the grid system that the application uses overall, and we must also make sure the relevant buttons appear on mobile screens.

bilinkis commented on _pages/profile/submit/submit-profile/finalize-tab.js:
```diff
@@ -83,7 +84,7 @@ class _FinalizeTab extends React.Component {
+                    <Link href="https://ethermail.io/">ethermail.io</Link>. Make sure
                     to check it while submission is being processed.
                   </Paragraph>
                 </>
```
Color of the name element should be white instead of black

santisiri replied:
Will fix on a subsequent pull.

santisiri replied:
Update: I will remove this and have a proper setup once Etheremail effectively deploys their PoH integration.

mmiguerodriguez commented on _pages/profile/[id]/submission-details-accordion.js:
```diff
@@ -125,7 +125,7 @@ export default function SubmissionDetailsAccordion({ submission, contract }) {
-        panelSx={{ paddingX: 0 }}
+        panelSx={{ paddingX: 0, borderRadius: 12 }}
         panel={
           <Evidence
```
Nice to have: Add margin between evidence text and corresponding card.

santisiri replied:
Fixed with the latest push: https://github.com/bilinkis/proof-of-humanity-web/pull/3/commits/0a332745ed16a93008aa554daaa8ce5ba0b5480c 

Hope I understood you correctly, looks better imho now:
<img width="265" alt="Screen Shot 2022-11-21 at 5 26 23 PM" src="https://user-images.githubusercontent.com/2153166/203109324-2c7f7a90-9b4d-41c1-85e8-f43d2c68656a.png">

mmiguerodriguez commented on _pages/profile/submit/submit-profile/finalize-tab.js:
```diff
@@ -75,6 +75,7 @@ class _FinalizeTab extends React.Component {
           <Col span={24}>
```
`About me` on the finalize-tab is black and feels weird. Text should be white

santisiri replied:
Now I realize what you and @bilinkis meant. Here's a quick fix: https://github.com/OpenProofOfHumanity/proof-of-humanity-web/pull/6/commits/8579239a781d3b9df5ec8756830c3b22de1d0d66

And here's how it looks now: 
<img width="941" alt="Screen Shot 2022-11-21 at 5 14 14 PM" src="https://user-images.githubusercontent.com/2153166/203104466-55da4445-6650-4623-b0a0-3b13539c9814.png">

mmiguerodriguez reviewed (changes requested):
LGTM. Good work.
I'm still not completely sure about the green/blue colors on the submit-profile/evidence tab. 
Only changes needed @ `finalize-tab -> about me` .

santisiri commented:
On the colors, here's a test of the palette we are using. Main color remains as the yellow trend since that has been the predominant color of PoH.. but we need secondary and tertiary colors to convey these are other smart contracts / environments we are interacting with in the long run. 

TL;DR: there's a plan. 
<img width="515" alt="Screen Shot 2022-09-24 at 7 42 36 PM" src="https://user-images.githubusercontent.com/2153166/203094930-3febe1cc-49e4-4a2b-aed2-f626dda9f68d.png">

santisiri committed:
fix: white text on finalize tab on the upload wizard

santisiri committed:
Merge pull request #6 from OpenProofOfHumanity/style

fix: white text on finalize tab on the upload wizard

santisiri committed:
fix: adding margin to evidence header

santisiri committed:
Merge pull request #7 from OpenProofOfHumanity/style

fix: adding margin to evidence header

santisiri committed:
fix: inversed colors of icons that were using black

santisiri committed:
fix: temporarily removed ethermail integration until they deploy

santisiri committed:
Merge pull request #8 from OpenProofOfHumanity/style

Last improvements in style

santisiri commented:
@bilinkis @mmiguerodriguez I think I finished addressing your concerns. Let me know if there's anything else I should have a look. Thanks for your help!

mmiguerodriguez reviewed (approved):
LGTM! Great step improving the current UI/UX.
Hopefully in the future we can have a revamp/fixing in which we can address package versioning/compatibility issues we may not be tackling right now.

bilinkis reviewed (approved):
LGTM.

bilinkis committed:
fix: header url width
